libs/perun/services/src/lib/init-auth.service.ts

```diff
@@ -40,8 +43,12 @@ export class InitAuthService {
     return this.authzService.getPerunPrincipal()
       .toPromise()
       .then(perunPrincipal => {
-        this.storeService.setPerunPrincipal(perunPrincipal);
-        this.authResolver.init(perunPrincipal);
+        if (perunPrincipal.user === null) {
+          this.dialog.open(UserDontExistDialogComponent, {height: '80px'});
```

Comment:
Why is there a height limit? This height causes a scrollbar to appear.

Reply:
without it, dialog had lot of empty space under text. It didn't show scrollbar on my screen, but i didn't check smaller resolutions.

Reply:
i will play a bit with it and find solution

Reply:
To reduce the padding or margin of an element, just change the CSS. Or, you can even create a new class with only those CSS attributes that you want to use.

Reply:
This should not be hard at all. It is just a matter of a couple of minutes to write a CSS class and apply it.